CHANGELOG.md

```diff
@@ -130,3 +395,5 @@ The changelog of JabRef 2.11 and all previous versions is available as [text fil
 [4.0-beta]: https://github.com/JabRef/jabref/compare/v3.8.2...v4.0-beta
 [2.11.1]: https://github.com/JabRef/jabref/compare/v2.11...v2.11.1
 [JavaFX]: https://en.wikipedia.org/wiki/JavaFX
+
+>>>>>>> f7f5841bdb1648947cbd257f1128fa46c0c7b524
```

Comment:
You still have this conflict markers in the file. The easiest solution to fix this would be to just copy the current changelog.md from master and paste it in your file locally and then commit the change.
So then git should be able to recognize that it didn't change